bylaws/2-cf-membership.md

```diff
@@ -6,12 +6,19 @@ weight: 2
 
 The Commonhaus Foundation (CF) is a membership organization. The following section outlines the different types of membership and the process for becoming a member.
 
+- [Eligibility](#eligibility)
 - [Project Leaders and Code Owners](#project-leaders-and-code-owners)
 - [General Members](#general-members)
 - [Membership Dues and Renewal](#membership-dues-and-renewal)
 - [Inactive or Retired Members](#inactive-or-retired-members)
 - [Suspension and Revocation of Membership](#suspension-and-revocation-of-membership)
 
+## Eligibility
+
+All Commonhaus Foundation members must be natural persons.
+
+Legal entities may nominate representatives to become members as outlined in the following sections.
```

Comment:
This is a good question: 

- If a legal entity nominates a representative, that may be a nominated member of an advisory board.

- A project could nominate or identify the leads that represent them as Commonhaus Members (according to their leadership/governance practices). 

We do need to define what a project is (see below re: Project Leaders and Code Owners). It is not a member itself, but it does represent a self-governing group of people. Project leads are members, but I think I would encourage others to join as individuals. An opt-in signals a willingness to participate and vote, etc.

Reply:
I added a paragraph to define projects here: https://github.com/commonhaus/foundation-draft/pull/52/files#diff-2ef9fee12127c3d1e9cb5c256a855a31a0ad5ee5d311f8ddf3feeff20530a326

That could imply a bit of a restructure to the top of this: 

Projects (not members themselves, but Project Leaders and Code Owners are defined within the context of a Project)

Then Members as individuals (natural persons) that join voluntarily. I am not sure what the best way to say that is. But if you (Yoann) join the CF, you're doing so as yourself, not as your employer. I would also add that members are expected to behave and vote in a manner that is (in that individual's opinion) in the best interest of the foundation and the foundation's projects.

Sponsoring organizations may nominate representatives to the Advisory Board to provide input and represent their views through that forum.

Reply:
> I am not sure what the best way to say that is. But if you (Yoann) join the CF, you're doing so as yourself, not as your employer

I think it's a bit more complicated than that.

I lead Hibernate Search, and my employer pays me for that. Everything I contribute to Hibernate Search is currently the property of my employer, and I'm contractually forbidden from going against my employer's instructions or interests when it comes to anything related to my job, even on my free time (even if my employer's internal policy is more generous and flexible that this, it's still what I committed to by signing my contract). This includes contributions to Hibernate Search of course, but also, arguably, what I would vote on as a member of this foundation... ?

So, yes, I join as an individual, but my employer can still control how I vote in this context.

I guess this all falls under the category of conflicts of interests, which I think is already covered elsewhere by requiring members to declare their conflicts, though I doubt it will be enforceable for anyone beside CFC members and project leads.

The fact that an organization can just have its employees join as "plain" members and have them vote _en masse_ is probably something we'll have to live with. This does represent a risk of takeover (by gradually voting in CFC members from that company), but it's probably mitigated elsewhere (max. representation of any given legal entity in the CFC, ...).

Reply:
So. This is a question that I will clarify with a lawyer, but I think we need to state this plainly: 

- Projects that have been donated to CF are owned by CF.
- CF Members are individuals, not corporate entities.
- Corporate contacts can lay a claim on your work and your time. They can determine whether or not you spend time and effort working on an open source project that they do not own. 

They can not control your vote.

If you are a nominated project lead... your projects interests and your employers interests will probably align. And yes, disclosing that relationhip would be normal practice. But a general, opt-in CF member is their own person, not their employer.

> The fact that an organization can just have its employees join as "plain" members and have them vote en masse is probably something we'll have to live with. This does represent a risk of takeover (by gradually voting in CFC members from that company), but it's probably mitigated elsewhere (max. representation of any given legal entity in the CFC, ...).

Yep. We do have the override clauses that I just added, and I suppose we should think about how that works. For example, if vote to override a CFC decision is held, there is a maximum number of votes that can come from any single legal entity to specifically handle that case.

Reply:
> They can not control your vote.

That's hard to put into words, so indeed a lawyer's opinion would help. First we can't tell employers what they should do if they do not have any kind of contract with the CF, so all I could find was a commitment from organizations that are part of the advisory board. Second, we need that commitment to be something realistic and provable, i.e. "organizations can't influence a member" is unrealistic since an employer will necessarily influence their employees one way or another.

I gave it a shot anyway and updated the PR, trying to merge your suggestions here with those from https://github.com/commonhaus/foundation-draft/discussions/51#discussioncomment-8138585. Let me know what you think?

Reply:
I've sent a quick question off to a lawyer, but... if someone feels their employer is requiring a vote in a direction that disproportionately benefits their company (rather than benefiting and aligning with the common interest of projects and members), the conflict of interest policy suggests that member should abstain.